.github/workflows/tests.yml

```
@@ -30,4 +30,4 @@ jobs:
       shell: bash -l {0}
       run: |
         micromamba activate TEST
-        python -m pytest -rxs --cov=pyobis ./pyobis
+        python -m pytest -rxs --cov=pyobis ./pyobis --vcr-record=none
```

I like to add this comment for my future self and future contributors. 
No strong opinion if you do not want to add that here and/or if you want to have it in a dev documentation elsewhere.

```suggestion
        # use --vcr-record=all when recording new cassettes. We never record on CIs though
        python -m pytest -rxs --cov=pyobis ./pyobis --vcr-record=none
```

---

Thank you so much for the suggestion! This is a valuable addition and I think it makes the new change more understandable without needing to search for the commit messages or this PR.